src/Xamarin.Android.Build.Tasks/MSBuild/Xamarin/Android/Xamarin.Android.AvailableItems.targets

```diff
@@ -86,6 +86,8 @@ This item group populates the Build Action drop-down in IDEs.
     <!-- Any .NET 6 project -->
     <ItemGroup Condition=" '$(UsingAndroidNETSdk)' == 'true' ">
       <LibraryProjectZip  Include="@(AndroidLibrary)" Condition=" '%(AndroidLibrary.Extension)' == '.aar' and '%(AndroidLibrary.Bind)' == 'true' " />
+      <!-- .NET 6 application projects -->
+      <InputJar           Include="@(AndroidLibrary)" Condition=" '$(AndroidApplication)' == 'true' and '%(AndroidLibrary.Extension)' == '.jar' and '%(AndroidLibrary.Bind)' == 'true' " />
```

Comment:
Given the presence of (new) line 93, would it be "better" if we made an explicit ".NET 6 applications" group and put it there?

```xml
<ItemGroup Condition=" '$(UsingAndroidNETSdk)' == 'true' "> … </ItemGroup>
<ItemGroup Condition=" '$(UsingAndroidNETSdk)' == 'true' And '$(AndroidApplication)' == 'true' ">
  <InputJar           Include="@(AndroidLibrary)" Condition=" '%(AndroidLibrary.Extension)' == '.jar' and '%(AndroidLibrary.Bind)' == 'true' " />
</ItemGroup>
<ItemGroup Condition=" '$(UsingAndroidNETSdk)' == 'true' And '$(AndroidApplication)' != 'true' ">…</ItemGroup>
```

Reply:
I moved it around. 👍